src/containers/FollowButton/index.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { followAction, unfollowAction, fetchAction } from "../FollowUnfollow/state/actions";
+
+class FollowUnfollowButton extends React.Component {
+  componentDidMount = () => {
+    const { fetch, username } = this.props;
+    fetch(username);
+  };
+
+  handleFollow = () => {
+    const { follow, username } = this.props;
+    follow(username);
+  };
+
+  handleUnfollow = () => {
+    const { unfollow, username } = this.props;
```

Comment:
Add test for this function check if the prop is called with the correct arguments

Reply:
Noted